lib/meadow/data/metadata.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Meadow.Data.Metadata do
+  @moduledoc """
+  Descriptive metadata associated with Work records.
+  """
+
+  import Ecto.Changeset
+  use Ecto.Schema
+
+  embedded_schema do
+    field :title
+    field :accession_number
```

Comment:
I'm thinking we should require uniqueness at the db level as well probably. Can we do this from an embedded schema?